src/main/java/domain/Car.java

```diff
@@ -20,11 +20,11 @@ public static Car of(String name, int distance) {
 
 
     public void move() {
-        distance.increase();
+        this.distance = new Distance(distance.getValue() + 1);
```

Comment:
```suggestion
        this. distance = this.distance.increase()
```

getValue를 써서 계산하는것 보다 메세지를 던지는게 더 자연스러워 보일것 같습니다.

Reply:
Distance를 값 객체로 바꾸었는데 값 객체면 불변이라서 스스로 값을 못 증가 시키지 않을까요?
제가 잘못 생각하고 있는거라면 말씀 부탁드립니다!

Reply:
DM으로 해소된 것 같으니 스레드를 닫겠습니다.